### DISCUSSION TITLE
customize dashboard

### DISCUSSION THREAD

**Discussion by asifrana1972** [horilla-opensource/horilla#201] [General]
Hi, can we customize dashboard? user specific?

**Comment by horilla-opensource** [comment]
> Hi, can we customize dashboard? user specific?



> Hi, can we customize dashboard? user specific?

Hi @asifrana1972 ,
The users can make use of the dashboard selection option in the quick actions buttons to toggle the dashboard tiles that they require.
1.
<img width="1470" alt="Screenshot 2024-06-04 at 10 35 10 AM" src="https://github.com/horilla-opensource/horilla/assets/131998600/855b6c10-ca2b-41f0-a625-fabe38acdb3b">

2.
<img width="1470" alt="Screenshot 2024-06-04 at 10 35 44 AM" src="https://github.com/horilla-opensource/horilla/assets/131998600/31b6649e-f970-4cd4-ac62-5ce864ab78b6">

3.
<img width="1470" alt="Screenshot 2024-06-04 at 10 37 12 AM" src="https://github.com/horilla-opensource/horilla/assets/131998600/2d8c6874-b2cc-4d3c-91a1-48df94661bd2">

**Reply by sharukhkhan234** [reply]
Hi!
we want some charts not to be displayed to users on the user dashboard, only the admin can see all the charts on the dashboard. for instance, some users should not be allowed to see this chart on the dashboard.
![employee_chart](https://github.com/horilla-opensource/horilla/assets/157000111/cdec27f9-0535-4fc6-bb93-a3fc5d72a6ca)

Regards,

**Comment by horilla-opensource** [comment]
Hi @sharukhkhan234 ,
The employee details display to all the users is a access right issue in the code. We will add a fix for it soon.
All other charts will display only the data to which the user has access to except for the offline and online employees chart.

**Comment by asifrana1972** [comment]
casual leave not updated , still on 1
![casual leave](https://github.com/horilla-opensource/horilla/assets/171243077/4c44aca0-0f97-45b7-8ed9-204029735118)

**Comment by asifrana1972** [comment]
Hi, can we disable on screen check-in/check-out. Users are allowed to check-in from biometric device only.

**Reply by horilla-opensource** [reply]
There is no option to remove the check-in/check-out button from the view directly.
You'll have to remove the html code for the button from its template code.

**Reply by sharukhkhan234** [reply]
can you share the location of the HTML file(in which app this file is located)

**Reply by horilla-opensource** [reply]
The button is contained in the index.html file (templates > index.html ). You can search for the element with class **_hr-Check-In-out-text_** to easily identify the element.

**Reply by sharukhkhan234** [reply]
Done,

thanks!

**Comment by asifrana1972** [comment]
I have not seen any field of employee id card number and its expiry date.
There must be an auto generated email on expiry.

On Wed, Jun 5, 2024, 12:48 PM sharukhkhan234 ***@***.***>
wrote:

> Done,
>
> thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/horilla-opensource/horilla/discussions/201#discussioncomment-9674728>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BI2PMRPEVXZMFTBOR2NSO6TZF27EDAVCNFSM6AAAAABIXWUYMSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TMNZUG4ZDQ>
> .
> You are receiving this because you modified the open/close state.Message
> ID: <horilla-opensource/horilla/repo-discussions/201/comments/9674728@
> github.com>
>

-- 

This email and any files transmitted with it are confidential and intended 
solely for the use of the individual or entity to whom they are addressed. 
If you have received this email in error please notify the system manager. 
This message contains confidential information and is intended only for the 
individual named. If you are not the named addressee you should not 
disseminate, distribute or copy this e-mail. Please notify the sender 
immediately by e-mail if you have received this e-mail by mistake and 
delete this e-mail from your system. If you are not the intended recipient 
you are notified that disclosing, copying, distributing or taking any 
action in reliance on the contents of this information is strictly 
prohibited.

**Reply by horilla-opensource** [reply]
There is expiry for the contract of a employee.

**Reply by asifrana1972** [reply]
But I am talking about employee national id card number which is mandatory field in HR.

**Reply by horilla-opensource** [reply]
Yes, we do have that also. The national id card which can be uploaded to the documents of the employee has an expiry in the document view.
